src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -142,7 +195,7 @@
     service :- [ServiceInfo]
     level :- ServiceStatusDetailLevel
     service-status-version :- (schema/maybe schema/Int)]
-    (let [status (if (nil? service-status-version)
+   (let [status (if (nil? service-status-version)
```

Comment:
If the let is moving then the entire binding vector & body need to shift as well, to maintain indentation. That should probably be done in a separate commit.